Resources/doc/reference/installation.rst

```diff
@@ -55,10 +56,15 @@ You will need to configure SonataArticleBundle's entities, if you plan to use th
                 article:  Application\Sonata\ArticleBundle\Entity\Article
                 fragment: Application\Sonata\ArticleBundle\Entity\Fragment
 
-            fragments: # this is currently in development, hope we'll get this soon :)
+            fragments:
                 sonata.article.fragment.title:
                 sonata.article.fragment.text:
 
+.. note::
+
+    We plan to improve the fragments available to allow a configuration for each article type.
+    For example, you will want the fragment 'Comments' only on articles of type 'Blog'.
+
```

Comment:
Indeed at first I thought of implement you first option (a simple array with all fragments) but then i decide to make it more extendable for the futur, when we will add the article type feature. That's why I prefer to keep it like this.

I haven't thought at all of a fragment provider and it seems quite complicated, I prefer your first comment option, seems quite simpler to configure. 

But maybe your provider will be more extensible, as I don't know how to handle types for article properly. It's hard for me to choose option or the other.

Since this bundle is still a work in progress, have BC Break later will not be an issue for the moment ?